docs/source/index.rst

```diff
@@ -2,26 +2,28 @@
 Chainer -- A flexible framework of neural networks
 ==================================================
 
-This is the `Chainer <https://chainer.org>`_ documentation.
+Welcome to the `Chainer <https://chainer.org>`_ documentation.
+
+* Certified 99% Python code!
+* Define-by-run approach for flexibility and understandable errors
+* Standard Numpy syntax
+* NVIDIA GPU acceleration, thanks to `CuPy <https://cupy.chainer.org>`_
 
 .. toctree::
-   :maxdepth: 1
+   :maxdepth: 2
+   :caption: Chainer Documents
 
-   tutorial/index
+   install
+   guides/index
+   examples/index
    reference/index
 
 .. toctree::
    :maxdepth: 1
-   :caption: Development
+   :caption: Other Stuff
```

Comment:
Does this stuff belong here?

Reply:
I think so. Have a better place to put it?

Reply:
Oh, what I meant was that it sounds a bit casual inside a documentation. Not a string opinion.